modules/tensor_mechanics/src/userobjects/CrystalPlasticityStateVarRateComponentGSS.C

```diff
@@ -34,7 +34,7 @@ CrystalPlasticityStateVarRateComponentGSS::calcStateVariableEvolutionRateCompone
 
   Real r = _hprops[0];
   Real h0 = _hprops[1];
-  Real tau_init = _hprops[2];
+//  Real tau_init = _hprops[2];
```

Comment:
Well my point is: Why is it not being used? It's in the middle of this vector and you're just ignoring it? Is this a design issue?